core/build.gradle.kts

```diff
@@ -51,8 +50,6 @@ kotlin {
         target("androidNativeX86")
         target("androidNativeX64")
          */
-        // Tier 4 (deprecated, but still in demand)
-        target("linuxArm32Hfp")
```

Comment:
Needs to be restored.

Reply:
Fixed

Reply:
I still see it missing here. I believe we can only restore it after kotlinx-serialization with this target is released. I left this comment not to forget about it.